common/src/test/java/com/microsoft/identity/common/internal/activebrokerdiscovery/BrokerDiscoveryClientTests.kt

```diff
@@ -70,7 +70,7 @@ class BrokerDiscoveryClientTests {
                         )
                         returnBundle.putString(
                             BrokerDiscoveryClient.ACTIVE_BROKER_SIGNATURE_HASH_BUNDLE_KEY,
```

Comment:
This constant should probably be renamed, but that can be done in a diff PR (by either you or me)